charts/gateway/templates/deployment.yaml

```diff
@@ -63,6 +63,117 @@ spec:
       {{- if .Values.initContainers }}
       initContainers: {{- toYaml .Values.initContainers | nindent 12 }}
       {{- end }}
+      {{ if and (.Values.otk.enabled) (not .Values.database.enabled) }}
```

Comment:
Make the otk init container to be the first in order, so that customers can overwrite the otk bundles if needed.

Reply:
done